wouso/games/teamquest/models.py

```diff
@@ -12,11 +15,32 @@ def is_head(self):
             return False
         return self.group.head == self
 
+class TeamQuestLevel(models.Model):
+    questions = models.ManyToManyField(Question)
+    quest = models.ForeignKey('TeamQuest', null=True, blank=True, related_name='levels')
+    bonus = models.IntegerField(default=0)
+
+    @classmethod
+    def create(cls, quest, bonus, **kwargs):
```

Comment:
Replace **kwargs with questions. You usually obtain a list of questions (e.g. by querying the db) and you want to pass that to the create method.